src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
-}
+  const rentCar = 40;
+  const threeDay = 3;
+  const sixDay = 6;
```

Comment:
Use a more descriptive name for the 'sixDay' constant, like 'maxDaysForSmallDiscount'.